angular-primeng-app/src/styles.scss

```diff
@@ -41,18 +41,27 @@ a {
 }
 
 article {
-  pre {
-    line-height: 2.2rem;
-    padding: 1rem 1.5rem 1rem;
-    border-radius: 0.3rem;
-    position: relative;
-
-    .hljs-keyword {
-      color: #f92672;
+  .content {
+    p {
+      img {
+        width: 100%;
+        margin: 0 auto;
```

Comment:
that's good, I am glad that you removed the redundant `display: block` property, we should probably remove it for other versions too 👍

Reply:
thank you @monacodelisa I value code review from you